tests/integration-tests/tests/test_efa/test_efa/test_efa/osu_submit.sh

```diff
@@ -2,4 +2,4 @@
 set -e
 
 module load openmpi
-mpirun --map-by ppr:1:node /shared/osu-micro-benchmarks-5.4/mpi/pt2pt/osu_latency &> /shared/osu.out
+mpirun -N 1 -np 2 /shared/osu-micro-benchmarks-5.4/mpi/pt2pt/osu_latency
```

Comment:
Did you try this with both sge and slurm?

Reply:
nope